scenarios/reproducers/dt-nfv-ovs-dpdk-sriov-hci.yml

```diff
@@ -0,0 +1,128 @@
+---
+cifmw_architecture_scenario: "nfv-ovs-dpdk-sriov-hci"
+cifmw_arch_automation_file: nfv-ovs-dpdk-sriov-hci.yaml
```

Comment:
It will be no longer needed as https://github.com/openstack-k8s-operators/ci-framework/pull/2267 gets merged first

Reply:
done